salt/metalk8s/defaults.yaml

```diff
@@ -31,6 +32,9 @@ repo:
   kubernetes:
     version: latest
 
+network:
+  control_plane: "10.0.0.0/8"
```

Comment:
This now shows up twice?

If this is a 'default', I disagree. We can't come up with a sensible 'default' for this value. You picked `10.0.0.0/8`. Why not `192.168.123.43/30`? Or any other random network.

Reply:
I think i will remove this from default and just fail is it's not specified in the pillar. And just give a default pillar with 10.0.0.0/8